integration-tests/index.test.js

```diff
@@ -660,3 +612,241 @@ query ntee {
 
   instance.close();
 });
+
+describe('authed mutations', () => {
+  const db = dbFactory();
+
+  let user, user2;
+
+  beforeAll(async () => {
+    user = await db.User.create({
+      name: 'test user',
+      email: 'good@user',
+    });
+    user2 = await db.User.create({
+      name: 'test user 2',
+      email: 'another@user2',
+    });
+  });
+
+  afterAll(async () => {
+    await db.Pdf.destroy({ where: {} });
+    await db.User.destroy({ where: {} });
+  });
+
+  test('add a grant', async () => {
+    const { uri, instance } = await createServerInstance();
+
+    const client = new GraphQLClient(uri, {
+      headers: {
+        'X-Auth-Token': 'good user',
+      },
+    });
+
+    const twoOrgs = await client.request(`
+  query twoOrgs {
```

Comment:
Dumb question, is the left align necessary here? makes it a little tough to skim but NBD

Reply:
i'm very open to changing the indentation for these blocks!

Reply:
Yeah! I wondered if it was necessary for parsing or a convention for graphql but I'm not familiar. If it's not having it indented I think would help.